config/main.py

```diff
@@ -115,6 +116,16 @@ def get_command(self, ctx, cmd_name):
 # Helper functions
 #
 
+# Execute action per NPU instance for multi instance services.
+def execute_asic_instance(inst, event, service, action):
```

Comment:
Function name isn't descriptive enough. I believe it should include 'systemctl'.

Reply:
Updated the function name.